### ISSUE TITLE
Problems in v3.4.0

### ISSUE THREAD

**Issue by shanalikhan** [shanalikhan/code-settings-sync#952]
Issues in v3.4.0

1. `lastUpload` not updated
2. #316 is pending.

**Comment by shanalikhan** [comment]
@arnohovhannisyan 
I have fixed the critical issue, `lastUpload` was not updating as the code was removed after update process to update the settings.

I can still confirm, once you upload the changes and hit upload second time, although the settings are same it still upload once again. (#316) , https://github.com/shanalikhan/code-settings-sync/pull/944#issuecomment-509160395


Can you pull the latest changes by cloning or `git clean` and `git reset` and test it again.
My env. 

Ubuntu  + Latest Code Standard version

**Comment by arnohovhannisyan** [comment]
@shanalikhan 

These are my exact steps:
```sh
cd /tmp
git clone git@github.com:shanalikhan/code-settings-sync -b v3.4.0
cd code-settings-sync
yarn
yarn compile
rm -rf ~/Downloads/code-1/data/*
~/Downloads/code-1/bin/code --extensionDevelopmentPath="`pwd`"
```

1. Log in and download settings
2. Try to upload without changing anything
  a. The extension warned me that the gist contents were either newer or identical to local content.
3. Change some of my settings and upload again
  a. No warning 
4. Upload again without changing anything
  a. Get the same warning

Note: `~/Downloads/code-1` is a portable version of VSCode. I'm deleting the data to simulate a fresh install.

**Comment by shanalikhan** [comment]
Try reproducing using these steps:

1. Reset or fresh Settings Sync installation and vscode Standard version
2. Login via Github and select the most recent [GIST](https://gist.github.com/shanalikhan/e86911443d3f81be6a66699603aee6dc). ( I'm using the attached one)
3. Upload the changes
4. Clear the summary output and hit upload again.
5. You will see it will upload again instead of showing the expected Message of not to upload due to same settings in Github.

**Comment by shanalikhan** [comment]
Can you improve the WIKI.

I suggest
We need to remove the `Upload Your Settings For the first time` and instead use the process for the following

1. Configure the Settings Sync  ( with images attached )
   - Tell user Settings Sync require two things to setup i.e Github Token ID and GIST ID
   - Page will be opened automatically on Code start.
   - Process to get the Github Token
   - New GIST will be created automatically if there is no GIST connected with user account or It will open new page with the GIST lists and allow user to select any GIST or skip if user wants to upload to new gist.

2. Upload the Settings ( with images attached )
   - Pre-req:  The Settings Should be configured or the Configuration Settings will be opened automatically to configure the Settings Sync. 
   - It will upload the settings and let user know the summary details of each files and extensions.

3. Download the Settings ( with images attached )
   - Pre-req:  The Settings Should be configured or the Configuration Settings will be opened automatically to configure the Settings Sync. 
   - It will download the settings and let know the summary details of each files and extension downloaded

**Comment by arnohovhannisyan** [comment]
@shanalikhan Currently, the wiki and `README.md` have pretty similar contents, but they both have to be updated when something changes. I propose that we just put a link to the wiki in `README.md` so we only have to maintain the wiki. What do you think?

**Comment by shanalikhan** [comment]
@arnohovhannisyan 

1. Have you tried those steps regarding auto Upload ? whats is the status.
2. We should only keep the Initial Setup or the User Manual inside the `README.md` and use the Github Wiki for advance guidelines ( e.g. sharing settings, using hosts and sync pragmas ) and for developer usage.

**Comment by arnohovhannisyan** [comment]
@shanalikhan 

> Have you tried those steps regarding auto Upload? what's is the status.

I have tried those steps many times and I get the expected result every single time. Can you try reproducing with a portable version of VSCode?

Get the portable version: 
```sh
cd /tmp
wget https://az764295.vo.msecnd.net/stable/2213894ea0415ee8c85c5eea0d0ff81ecc191529/code-stable-1562627471.tar.gz
tar xzf code-stable-1562627471.tar.gz
mkdir VSCode-linux-x64/data
```

Run the extension:

```sh
cd /tmp
git clone git@github.com:shanalikhan/code-settings-sync -b v3.4.0
cd code-settings-sync
npm install && npm run compile
/tmp/VSCode-linux-x64/bin/code --extensionDevelopmentPath="`pwd`"
```

**Comment by shanalikhan** [comment]
I have released the `v3.4.0`.

Let me know if there is any issue, i will release new version fixing this.
